### ISSUE TITLE
fetch remote data with websocket connection

### ISSUE THREAD

**Issue by Chellhooo** [nhn/tui.grid#788]
**Summary**
hi~ Datasource as below seems the only way to fetch remote data with service side pagination.  But if I want to update via websockets for real-time financial record, is there a way to fetch remote data and reset pagination myself with **function**?

```
import Grid from 'tui-grid';
const dataSource = {
  api: {
    readData: { url: '/api/read', method: 'GET' }
  }
};
const grid = new Grid({
  data: dataSource
});
```

**Version**
tui-grid: 4.6.1

**Comment by js87zz** [comment]
@Chellhooo 
In your case, It's better to use the `grid.resetData` API with `Client pagination` than using `dataSource` like as below.
But there is issue that miscalculating height(we are planned to fix it).  So, I recommend apply this after fixing the issue.
```js
const grid = new Grid({
  data: [],
  pageOptions: {
    useClient: true,
    perPage: 10
  }
});
// in your handler
() => {
  /*
  your code for grid data
  */
  grid.resetData(data);
}
```

**Comment by Chellhooo** [comment]
@js87zz Thanks for answer~ But it seams that I can't reset pagination like `total_count`? I sorry I can't find the api in [api doc](https://nhn.github.io/tui.grid/latest/Grid), do I miss document somewhere?

**Comment by js87zz** [comment]
@Chellhooo
`totalCount` is reset automatically using length of data as `resetData` parameter.
You can find the document [here](https://github.com/nhn/tui.grid/tree/master/packages/toast-ui.grid/docs)

**Comment by Chellhooo** [comment]
@js87zz ok. Thanks so much~

**Comment by js87zz** [comment]
@Chellhooo 
The bug(miscalculating height) has been fixed in `v4.8.0`
Thank you!
